TECHMANIA/Assets/Scripts/Components/ResourceLoader.cs

```diff
@@ -224,6 +224,10 @@ public static AudioClip GetCachedClip(string filenameWithoutFolder)
 
     private static bool IsEmptyFile(string fullPath)
     {
+        if (Paths.IsInStreamingAssets(fullPath))
+        {
+            return false;
```

Comment:
`// FileInfo.Length causes a read, which doesn't work on streaming assets on Android, so we simply bypass the test for streaming assets.`